src/app/onboarding/guards/onboarding.auth.guard.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, Router, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { Observable } from 'rxjs';
+
+import { AccountService } from '@shared/services/account/account.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class OnboardingAuthGuard implements CanActivate {
+  constructor(
+    private account: AccountService,
+    private router: Router,
+  ) {}
+
+  canActivate(
+    route: ActivatedRouteSnapshot,
+    state: RouterStateSnapshot): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
```

Comment:
The other advantage to wrapping the closing paren, return type, and opening brace onto its own line is it serves as a clearer separator between the parameter list and the body of the function.